astrocut/tests/test_utils.py

```diff
@@ -13,7 +16,51 @@
 with open(get_pkg_data_filename('data/ex_ffi_wcs.txt'), "r") as FLE:
     WCS_STR = FLE.read()
 
+
+def test_parse_size_input():
+    """Test that different types of input are accurately parsed into cutout sizes"""
+    # With scalar as input
+    cutout_size = utils.parse_size_input(5)
+    assert np.array_equal(cutout_size, np.array((5, 5)))
+
+    # With astropy quantity as input
+    cutout_size = utils.parse_size_input(10 * u.pix)
+    assert np.array_equal(cutout_size, np.array((10, 10)) * u.pix)
+
+    # With tuple as input
+    cutout_size = utils.parse_size_input((5, 10))
+    assert np.array_equal(cutout_size, np.array((5, 10)))
+
+    # With list as input
+    cutout_size = utils.parse_size_input([10, 5])
+    assert np.array_equal(cutout_size, np.array((10, 5)))
+
+    # With array as input
+    cutout_size = utils.parse_size_input(np.array((5, 10)))
+    assert np.array_equal(cutout_size, np.array((5, 10)))
```

Comment:
You can probably simplify this test by parametrizing over the inputs/outputs using `@pytest.mark.parametrize`